apps/chat-e2e/src/ui/selectors/folderSelectors.ts

```diff
@@ -1,5 +1,7 @@
 export const FolderSelectors = {
   folder: '[data-qa="folder"]',
+  rootFolder: () => `${FolderSelectors.folder}[property="root"]`,
+  childFolder: '[data-qa="folder"][property="child"]',
```

Comment:
please, apply the same approach as for rootFolder to avoid duplicates

Reply:
done